MAUI/Cartesian-Charts/Zooming-and-panning.md

```diff
@@ -79,6 +79,45 @@ chart.ZoomPanBehavior = zooming;
 
 {% endtabs %}
 
+### Directional Zooming
+
+The directional Zooming feature enhances your zooming experience by allowing you to zoom in and out in a specific direction. This feature is enabled by setting the [EnableDirectionalZooming]() property to `true` as shown in the following code sample. The default value of this property is false.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart>
+        ...
+        <chart:SfCartesianChart.ZoomPanBehavior>
+            <chart:ChartZoomPanBehavior EnablePinchZooming="True"
+                                        EnableDirectionalZooming="True"/>
+        </chart:SfCartesianChart.ZoomPanBehavior>
+        ...
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+    ...
+    ChartZoomPanBehavior zooming = new ChartZoomPanBehavior()
+    {
+        EnablePinchZooming=true,
```

Comment:
`EnablePinchZooming=true,`

Alignment not good.

Reply:
Addressed and Updated the changes.